document-generator-interface/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" 
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    
+    <modelVersion>4.0.0</modelVersion>
+  
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>document-generator</artifactId>
+        <version>unversioned</version>
+    </parent>
+  
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>document-generator-interface</artifactId>
+    <version>unversioned</version>
+    <name>document-generator-interface</name>
```

Comment:
Could we add a `description` tag? Just to say what it does? As a short description so when it gets built, the manifest for the jar has a nice description :)